packages/react-apollo/src/picker/Picker.jsx

```diff
@@ -5,11 +5,44 @@ import * as _ from 'lodash';
 import {PredefinedFragments, replaceFragmentsInDocument} from '@jahia/apollo-dx';
 import PropTypes from 'prop-types';
 
+const PickerItemsFragment = {
+    mixinTypes: {
+        applyFor: 'node',
+        variables: {
+            lang: 'String!'
+        },
+        gql: gql`fragment MixinTypes on JCRNode {
+            mixinTypes {
+                name
+            }
+        }`
+    },
+    isPublished: {
+        applyFor: 'node',
+        variables: {
+            lang: 'String!'
+        },
+        gql: gql`fragment PublicationStatus on JCRNode {
+            publicationStatus: aggregatedPublicationInfo(language: $lang) {
+                publicationStatus
+            }
+        }`
+    },
+    primaryNodeType: {
+        applyFor: 'node',
+        gql: gql`fragment PrimaryNodeTypeName on JCRNode {
+            primaryNodeType {
+                name
+            }
+        }`
+    }
+};
+
 class Picker extends React.Component {
     constructor(props) {
         super(props);
 
-        const {
+        let {
```

Comment:
ce qui est dans les props doit rester en const.

Reply:
Yes, but I just check in case "fragments" it's undefined to assign value to it, but yes I'll do this in default props value, thank you